src/pages/CropSelector/CropTable/RenderTableItems.js

```diff
@@ -36,54 +36,48 @@ const RenderTableItems = ({ showGrowthWindow, handleModalOpen }) => {
-                    className="col-auto pl-md-0 text-left"
-                    style={{
-                      display: 'flex',
-                      flexDirection: 'column',
-                      alignItems: 'flex-start',
-                      justifyContent: 'center',
+              <Grid
+                container
+                direction="column"
+                alignItems="center"
+                justifyContent="center"
+                height="100%"
+                spacing={0}
+              >
+                <Grid item xs={12}>
+                  <Button onClick={() => handleModalOpen(crop)}>
+                    <CropImage
+                      present={!!crop.thumbnail}
+                      src={crop.thumbnail ? crop.thumbnail : 'https://placehold.it/100x100'}
+                      alt={crop.label}
+                    />
+                  </Button>
+                </Grid>
+                <Grid item xs={12}>
+                  <Button
+                    variant="text"
+                    sx={{
+                      fontWeight: 'bold',
+                      textDecoration: 'underline',
+                      color: 'black',
+                      fontSize: '0.rem',
+                    }}
+                    onClick={() => {
+                      handleModalOpen(crop);
                     }}
                   >
-                    <div>
-                      <Typography variant="h7" sx={{ fontWeight: 'bold', textAlign: 'left' }}>
-                        {flipCoverCropName(crop.label)}
-                      </Typography>
-                    </div>
-                    <div>
-                      <Typography
-                        variant="body1"
-                        style={{
-                          color: 'gray',
-                          fontWeight: 'normal',
-                          fontStyle: 'italic',
-                          fontSize: 'small',
-                          textAlign: 'left',
-                        }}
-                      >
-                        {crop.scientificName && trimString(crop.scientificName, 25)}
-                      </Typography>
-                    </div>
-                    {/* <div className="col-12 p-md-0"> */}
-                    <div>
-                      <Typography variant="body1" style={{ color: 'gray', fontSize: 'small' }}>
-                        {crop.group}
-                      </Typography>
-                    </div>
-                  </div>
-                </div>
-              </div>
+                    {flipCoverCropName(crop.label)}
+                  </Button>
+                </Grid>
+                <Grid item xs={12}>
+                  <Typography sx={{ fontSize: '0.9rem' }}>
+                    {crop.scientificName && trimString(crop.scientificName, 20)}
+                  </Typography>
+                  <Typography variant="body1" style={{ color: 'gray', fontSize: '0.9rem' }}>
+                    {crop.group}
```

Comment:
crop.group seems to be undefined.

Reply:
yup, this is an api issue. trevor is working on it